src/services/AuthorizationRequestProcessor.ts

```diff
@@ -97,7 +97,7 @@ export class AuthorizationRequestProcessor {
 						event_name: "F2F_CRI_END",
 						...buildCoreEventFields(session, this.environmentVariables.issuer(), session.clientIpAddress, absoluteTimeNow),
 						extensions: {
-							previous_govuk_signin_journey_id: session.yotiSessionId,
+							previous_govuk_signin_journey_id: session.clientSessionId,
```

Comment:
Might be being a bit pedantic but just because there seems to be a bit of a focus on being able to trace changes, could we please update the ticket scope to include this event well?

Reply:
Good idea, just updated the ticket with it now saying to check all TxMA events